README.md

```diff
@@ -152,8 +152,9 @@ This module has _partial_ support for forwarding websockets by passing a
 A few things are missing:
 
 1. forwarding headers as well as `rewriteHeaders`
-2. support for paths, `prefix` and `rewritePrefix`
-3. request id logging
+2. request id logging
+
+The `prefix` and `rewritePrefix` options are also supported
```

Comment:
This seems insufficient documentation. Please explain what they are and how to use them with examples.

Reply:
@daofresh ping